Makefile-libs.Inc

```diff
@@ -89,17 +89,21 @@ endif
 $(PROJ_DIR)$(POSTRS_SETUP_ZIP):
 	curl -sSL $(CURL_OPTIONS) $(POSTRS_SETUP_URL_ZIP) -o $(PROJ_DIR)$(POSTRS_SETUP_ZIP)
 
-$(BIN_DIR)$(POSTRS_PROFILER_BIN):
+$(PROJ_DIR)$(POSTRS_PROFILER_ZIP):
 	curl -sSL $(CURL_OPTIONS) $(POSTRS_PROFILER_URL) -o $(PROJ_DIR)$(POSTRS_PROFILER_ZIP)
-	unzip -o -j $(PROJ_DIR)$(POSTRS_PROFILER_ZIP) -d $(BIN_DIR)
+
+$(BIN_DIR)$(POSTRS_PROFILER_BIN): $(PROJ_DIR)$(POSTRS_PROFILER_ZIP)
+	unzip -DD -o -j $(PROJ_DIR)$(POSTRS_PROFILER_ZIP) -d $(BIN_DIR)
 	chmod +x $(BIN_DIR)$(POSTRS_PROFILER_BIN)
 
-$(BIN_DIR)$(POSTRS_SERVICE_BIN):
+$(PROJ_DIR)$(POSTRS_SERVICE_ZIP):
 	curl -sSL $(CURL_OPTIONS) $(POSTRS_SERVICE_URL) -o $(PROJ_DIR)$(POSTRS_SERVICE_ZIP)
-	unzip -o -j $(PROJ_DIR)$(POSTRS_SERVICE_ZIP) -d $(BIN_DIR)
+
+$(BIN_DIR)$(POSTRS_SERVICE_BIN): $(PROJ_DIR)$(POSTRS_SERVICE_ZIP)
+	unzip -DD -o -j $(PROJ_DIR)$(POSTRS_SERVICE_ZIP) -d $(BIN_DIR)
```

Comment:
This change has a weird side-effect that already bothered me about the post service:

If I switch between branchew where different versions of the post service are used (as it is currently the case for some PRs) after the change calling `make get-libs` will be a no-op, if a different version is already present in the `build` directory.

Reply:
It's already behaving like that without my change because the _input_ to the targets like `$(BIN_DIR)$(POSTRS_SERVICE_BIN):` is the zip file. If both the binary and the zip exist, nothing will happen. It frustrates me as well. I'd like to fix it eventually too but it requires more work.

Reply:
I fixed this behavior in https://github.com/spacemeshos/go-spacemesh/pull/5532/commits/931871ccaf8be6b14dcd434cb95fc0dda988078d, please verify :)